src/main/java/com/wiley/autotest/selenium/elements/upgrade/should/GeneralListShould.java

```diff
@@ -40,6 +39,10 @@ public void haveText(String text) {
 
     }
 
+    @Override
+    public void haveText() {
```

Comment:
this method will do nothing for list and will not notify user about it. Please add the implementation for list accordingly.